Sources/Sentry/Public/SentryOptions.h

```diff
@@ -2,7 +2,15 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@class SentryDsn, SentrySdkInfo;
+@class SentryDsn, SentrySdkInfo, SentryTransactionSamplingContext;
+
+/** Function pointer for a sampler callback.
+ *
```

Comment:
```suggestion
/** 
 * Function pointer for a sampler callback.
 *
```

Reply:
`m`: Can we please put more info into this code doc, as here https://github.com/getsentry/sentry-java/blob/4620134a84de5e8a10a2811392f31941a10059cc/sentry/src/main/java/io/sentry/SentryOptions.java#L1293-L1301